src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -223,6 +230,132 @@ void refuseAttemptToMoveRequestBeingFulfilledFromOneOfTheTopPositionsWhenTlrEnab
+    assertThat(item.containsKey("volume"), is(true));
+    assertThat(item.containsKey("chronology"), is(true));
+    assertThat(item.containsKey("status"), is(true));
+    assertThat(item.containsKey("callNumber"), is(true));
+    assertThat(item.containsKey("copyNumber"), is(true));
+
+    assertThat(request.containsKey("loan"), is(true));
+    JsonObject loan = request.getJsonObject("loan");
+    assertThat(loan.containsKey("dueDate"), is(true));
+
+    assertThat(request.containsKey("requester"), Is.is(true));
+    JsonObject requester = request.getJsonObject("requester");
+    assertThat(requester.containsKey("lastName"), is(true));
+    assertThat(requester.containsKey("firstName"), is(true));
+    assertThat(requester.containsKey("middleName"), is(false));
+    assertThat(requester.containsKey("barcode"), is(true));
+    assertThat(requester.containsKey("patronGroup"), is(true));
+    assertThat(requester.containsKey("patronGroup"), Is.is(true));
+
+    assertThat(request.containsKey("proxy"), Is.is(true));
+    final JsonObject proxySummary = request.getJsonObject("proxy");
+    assertThat(proxySummary.containsKey("patronGroup"), Is.is(true));
+    assertThat(proxySummary.getString("patronGroupId"), UUIDMatcher.is(staffGroupId));
+    assertThat(proxySummary.getJsonObject("patronGroup").getString("id"),
+      UUIDMatcher.is(staffGroupId));
+
+    assertThat(request.containsKey("pickupServicePoint"), is(true));
+    final JsonObject pickupServicePoint = request.getJsonObject("pickupServicePoint");
+    assertThat(pickupServicePoint.containsKey("name"), is(true));
+    assertThat(pickupServicePoint.containsKey("code"), is(true));
+    assertThat(pickupServicePoint.containsKey("discoveryDisplayName"), is(true));
+    assertThat(pickupServicePoint.containsKey("pickupLocation"), is(true));
+  }
+
+  @Test
+  void shouldGetRequestQueueForInstanceSuccessfully() {
+    configurationsFixture.enableTlrFeature();
+
+    UUID isbnIdentifierId = identifierTypesFixture.isbn().getId();
+    String isbnValue = "9780866989427";
+    ItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet(
+      identity(),
+      instanceBuilder -> instanceBuilder.addIdentifier(isbnIdentifierId, isbnValue).withId(instanceId),
+      itemsFixture.addCallNumberStringComponents());
+    checkOutFixture.checkOutByBarcode(smallAngryPlanet);
+
+    ZonedDateTime requestDate = ZonedDateTime.of(2017, 7, 22, 10, 22, 54, 0, UTC);
+
+    requestsClient.create(new RequestBuilder()
+      .hold()
+      .titleRequestLevel()
+      .withInstanceId(instanceId)
+      .withNoItemId()
+      .withNoHoldingsRecordId()
+      .withPickupServicePointId(servicePointsFixture.cd1().getId())
+      .withRequestDate(requestDate)
+      .withRequesterId(usersFixture.steve().getId()));
+
+    JsonArray requests = requestQueueFixture.retrieveQueueForInstance(smallAngryPlanet.getInstanceId().toString()).getJsonArray("requests");
```

Comment:
Take the first element of the queue in this line instead of using the redundant variable. The same above.

Reply:
fixed